modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -321,9 +321,11 @@
  * A <b>shearing</b> transformation, sometimes called a skew, effectively
  * rotates one axis so that the x and y axes are no longer perpendicular.
  * <p>
- * Multiple transformations may be applied to a node by specifying an ordered
- * chain of transforms.  The order in which the transforms are applied is
- * defined by the ObservableList specified in the {@link #getTransforms transforms} variable.
```

Comment:
This block of comment sounds useful to me. It does talk very specifically about the order of transforms in the `getTransforms()` and is correct. How about keeping this block and then continue the new comment from `Custom...`